### PR TITLE
fix(lodash) type for sortedUniqBy

### PR DESCRIPTION
**Pull request by joacimastrom**
Just made minor changes shown below, but I get a huge diff due to eol and don't understand why. Editorconfig says ```end_of_line = lf``` and file is LF. Any ideas on how to fix that?

```
git diff -w
diff --git a/definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js b/definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js
index 4010684..b01fb61 100644
--- a/definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js
+++ b/definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js
@@ -358,7 +358,7 @@ declare module "lodash" {
     sortedLastIndexOf<T>(array: Array<T>, value: T): number;
     sortedLastIndexOf<T>(array: void | null, value?: ?T): -1;
     sortedUniq<T>(array?: ?Array<T>): Array<T>;
-    sortedUniqBy<T>(array?: ?Array<T>, iteratee?: ?(value: T) => mixed): Array<T>;
+    sortedUniqBy<T>(array?: ?Array<T>, iteratee?: ?ValueOnlyIteratee<T>): Array<T>;
     tail<T>(array?: ?Array<T>): Array<T>;
     take<T>(array?: ?Array<T>, n?: ?number): Array<T>;
     takeRight<T>(array?: ?Array<T>, n?: ?number): Array<T>;
@@ -1853,10 +1853,7 @@ declare module "lodash/fp" {
     sortedLastIndexOf<T>(value: T): (array: Array<T>) => number;
     sortedLastIndexOf<T>(value: T, array: Array<T>): number;
     sortedUniq<T>(array: Array<T>): Array<T>;
-    sortedUniqBy<T>(
-      iteratee: (value: T) => mixed
-    ): (array: Array<T>) => Array<T>;
-    sortedUniqBy<T>(iteratee: (value: T) => mixed, array: Array<T>): Array<T>;
+    sortedUniqBy<T>(iteratee: ValueOnlyIteratee<T>, array: Array<T>): Array<T>;
     tail<T>(array: Array<T>): Array<T>;
     take<T>(n: number): (array: Array<T>) => Array<T>;
     take<T>(n: number, array: Array<T>): Array<T>;
```

### CODE REVIEW

**Comment by jirutka** [comment]
That's not your fault; someone else ignored editorconfig and committed a CRLF garbage into this file.

Fortunately this is the only file in this repo with CRLF. I suggest to fix it in a separate commit and then apply your change.

**Comment by GAntoine** [comment]
@joacimastrom So the changes look fine, and I'm happy to see the EOL fixed. You'll need to add some specs to cover the changes in this PR before we can merge it in.

**Comment by joacimastrom** [comment]
@GAntoine Thank you for the reply! I kinda forgot about this but should be fixed now, a bit messy git log but it should be fine. 

Also added missing tests, don't know what other specs you need so if there is anything else just let me know!

**Comment by GAntoine** [comment]
Looks good! Any idea why one of the files is entirely marked as changed? I can't see any difference, even in whitespacing, and it's making it hard to review.

File: definitions/npm/lodash_v4.x.x/flow_v0.63.x-/test_lodash-fp-v4.x.x.js [hunk 1]
```diff
@@ -206,118 +209,130 @@ function square(n) {
...
+sortedLastIndexBy(
+  function(o) {
+    return o.x;
+  },
+  { x: 4 },
+  [{ x: 4 }, { x: 5 }]
+);
+sortedLastIndexBy("x", { x: 4 }, [{ x: 4 }, { x: 5 }]);
+
+/**
```
Comment (joacimastrom) [inline definitions/npm/lodash_v4.x.x/flow_v0.63.x-/test_lodash-fp-v4.x.x.js]: These tests were added for sortedUniq and sortedUniqBy, everything else is just my prettier formatting which I didn't think to turn off... 😬 @GAntoine

Comment (joacimastrom) [reply definitions/npm/lodash_v4.x.x/flow_v0.63.x-/test_lodash-fp-v4.x.x.js]: And for the lodash-file, I assumed it was due to the EOL. I practically just saved the file to fix that and committed, then made my changes in the next commit.

**Pull request merged**